File: UCFPT/22locals/local2022FinalRoundData/solutions/maxi/maxi.java
//
// 2022 UCF Local Programming Contest (Final Round) -- maxi
// Ali Orooji
//

import java.util.*;
import java.lang.Math;

public class maxi
{
	public static String num_str; // input number in string
	
	public static char num_char[]; // input number in char
	
	public static Scanner input;
	
   /* ********************************************************* */
	
   public static void main(String[] args)
   {
	  input = new Scanner(System.in);
	  
	  num_str = input.next();
	  solve();
	   
   }// end of main()

   /* ********************************************************* */
	
   public static void solve()
   {
	  num_char = new char[7];
	  
	  num_char = num_str.toCharArray();
	  Arrays.sort(num_char);
	  
	  // num_char is in increasing order; print it in reverse order
	  for (int k = num_str.length() - 1;  k >= 0;  --k)
		  System.out.print(num_char[k]);
	  
	  System.out.println();
	   
   }// end of solve()

   /* ********************************************************* */
 
}// end of class maxi

   /* ********************************************************* */

File: UCFPT/22locals/local2022FinalRoundData/solutions/maxi/maxi.cpp
// Arup Guha
// 9/6/2022
// C++ Solution for 2022 UCF Final Round Problem: Number Maximization

using namespace std;

#include <iostream>
#include <string>

int main(void) {

	// I'll use a frequency array for fun.
	int freq[10];
	for (int i=0; i<10; i++) freq[i] = 0;
	
	// Read as string.
	string s;
	cin >> s;
	
	// Update frequency array.
	for (int i=0; i<s.length(); i++)
		freq[s[i]-'0']++;
	
	// Print in reverse order.
	for (int i=9; i>=0; i--)
		for (int j=0; j<freq[i]; j++)
			cout << i;
	cout << endl;

	return 0;
}